src/app/pages/users/user/user.component.ts

```diff
@@ -640,6 +647,7 @@ export class UserComponent extends AbstractTabComponent implements OnInit {
 
   public closeDialog(saved: boolean = false) {
     if (this.inDialog) {
+      this.windowService.clearSearch();
```

Comment:
@LucasBrazi06 @ClaudeROSSI is this the right way to handle the current user ?

Reply:
No :)